staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd_test.go

```diff
@@ -95,17 +95,6 @@ func newStorage(t *testing.T) (customresource.CustomResourceStorage, *etcdtestin
 	return storage, server
 }
 
-// createCustomResource is a helper function that returns a CustomResource with the updated resource version.
-func createCustomResource(storage *customresource.REST, cr unstructured.Unstructured, t *testing.T) (unstructured.Unstructured, error) {
```

Comment:
I think I had added this to test generation in `TestGeneration` but the test never got added.

Instead of deleting this, can you add the test instead? There are integration tests which check if `metadata.generation` is incremented properly but would be nice to have a unit test too. :)

Something like here: https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/apps/replicaset/storage/storage_test.go#L160

Reply:
In addition to what is done in the above test, the feature gate will also need to be set:

https://github.com/kubernetes/kubernetes/blob/61cddc9a7fff5fcce043100165e502642b498134/staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go#L580-L581

Reply:
Done,Thanks for your review.